tests/js/spec/utils/utils.spec.jsx

```diff
@@ -234,3 +235,16 @@ describe('utils.projectDisplayCompare', function() {
     ]);
   });
 });
+
+describe('utils.stringifyRouteList', function() {
+  it('converts a routes list into a string', function() {
+    const routes = [
+      {path: '/one/'},
+      {path: '/two/'},
+      {path: undefined},
+      {path: '/three/'},
+      {path: '/:someKey/'},
+    ];
+    expect(stringifyRouteList(routes)).toEqual('one/two/three/:someKey/');
```

Comment:
Could we refactor this so that a leading `/` is kept? Reason being, we have other places where we do this and need the leading slash.

Reply:
Yeah, I'll just have it stick a leading `/`  on there

Reply:
9355fe3